### PR TITLE
First SA schema draft

### PR DESCRIPTION
**Pull request by emiliorighi**
First Sequence Feature definition
@egchristensen 
@larrybabb 
@ahwagner

### CODE REVIEW

File: sa.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  Gene:
+    properties:
+      transcripts:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "#/definitions/Transcript"
+      strand:
+        type: string
+        enum:
```
Comment (andrewyatz) [inline sa.yaml:30]: Strand is likely a reusable concept. Should be yanked out, but also has this been covered in VA? If not make sure this is the right way to encode

File: sa.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        - "-"
+      referenceGenome:
+        type: string
+    required:
+    - strand
+    additionalProperties: true
+  Transcript:
+    properties:
+      coding:
+        type: boolean
```
Comment (andrewyatz) [inline sa.yaml:41]: Would this apply to transcripts undergoing NMD? Is it a flag to say it has a protein product? What about active biomolecular products like lncRNA or miRNA?

File: sa.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-07/schema
+title: GA4GH-SA-Definitions
+type: object
+definitions:
+  SequenceFeature:
+    properties:
+      id:
+        type: string
+      location:
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/main/schema/vrs.yaml#definitions/Location"
```
Comment (andrewyatz) [inline sa.yaml:10]: Seems like we should have a pointer to a more stable URL? Is this a case for putting DOIs over schemas? Not sure if it helps or not ...

File: sa.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      seqOntologyRef:
```
Comment (andrewyatz) [inline sa.yaml:11]: Is this the SO term? Or the string? Is this component capable of modelling other ontological links? Is SO special?

**Review by andrewyatz — CHANGES_REQUESTED** [review]
First set of changes suggested. Mostly about extensibility and sharing between other models

File: sa.yaml (hunk 3 above)
Comment (egchristensen) [reply sa.yaml:10]: What's the best way to bring in the VRS location model into the SA spec @larrybabb @ahwagner? Does VRS generate DOIs/IRIs for its location object specs? Like, can we refer to the SequenceLocation object with an IRI?

File: sa.yaml (hunk 4 above)
Comment (egchristensen) [reply sa.yaml:11]: I think we would like to enforce SO terminology for sequence features (junctions, regions, etc.), but there are definitely other ontologies/terminologies/nomenclatures that we'll eventually want to support (HUGO for genes, possibly ECO for evidence codes, etc.) but I don't think they're directly relevant to the "SequenceFeature" object. 

Gene names seem to generally be stored separately from the sequence feature "gene". Do you feel that we need to have the gene named at this same level or are you ok with it being identified a bit deeper in the structure @andrewyatz?

File: sa.yaml (hunk 1 above)
Comment (egchristensen) [reply sa.yaml:30]: @lstein's [2020 GFF3 spec](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md) encodes strand this way @andrewyatz .

File: sa.yaml (hunk 2 above)
Comment (egchristensen) [reply sa.yaml:41]: @andrewyatz I'm still trying to think of a good way to represent sequence features that map to multiple types of sequence. I think we may want to explore @rrfreimuth's idea of MolecularState that he presented at the last connect meeting.

**Review by egchristensen — APPROVED** [review]

**Pull request merged**